src/components/Page/PageHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { usePageContext } from './Context';
+import classes from './PageHeader.module.css';
+import { PageIcon } from './PageIcon';
+
+export interface PageHeaderProps {
+  children?: React.ReactNode;
+  icon: JSX.Element;
```

Comment:
Is Icon really required? I think at least if we will use this in Altinn, we wont have an icon

Reply:
In authorization's frontend I've only found examples of Card with icon. Regardless of this I can make it nullable to streamline the component.